pages/am/3.x/user-guide/application/identity-providers/user-guide-client-identity-providers.adoc

```diff
@@ -0,0 +1,61 @@
+= Application Identity Providers
+:page-sidebar: am_3_x_sidebar
+:page-permalink: am/current/am_userguide_client_identity_providers.html
+:page-folder: am/user-guide
+:page-layout: am
+
+== Overview
+
+AM allows your application to use different identity providers. If you haven't configured your providers yet,
+visit the link:{{ '/am/current/am_userguide_create_identity_provider.html' | relative_url }}[Identity Provider^] guide.
+
+The application identity providers are separated into two sections:
+
+- The regular Identity Providers (called also *internal*) that operate inside and AM without redirecting to another provider
+- The Social/Enterprise Identity Providers that require an external service to perform authentication (usually via SSO)
+
+image::{% link images/am/current/graviteeio-am-userguide-application-identity-providers.png %}[]
+
+You can enable/disable them to include them within your authentication flow.
+
+== Priority
+
+[label label-version]#New in version 3.17#
+
+Identity provider priority enables processing authentication in a certain order. It gives more control over the authentication flow by deciding which provider should evaluate credentials
+first.
+
+In order to change the priority of the providers:
+
+ - Make sure your provider is *selected*
+ - Simply *drag-and-drop* the providers
+ - Save your settings
+
+== Selection rules
+
+[label label-version]#New in version 3.17#
+
+Identity provider selection rules also give you more control over the authentication via link:{{ '/apim/3.x/apim_publisherguide_expression_language.html' | relative_url }}[Expression Language^]
```

Comment:
We now have a page for EL on AM documentation. This has been merged into the `am_3.17` branch
so link should be `am/current/am_userguide_expression_language.html`

https://github.com/gravitee-io/gravitee-docs/blob/am_3.17/pages/am/3.x/user-guide/expression-language/user-guide-expression-language.adoc